### PR TITLE
Doc: Added raise_error middleware configuration

### PR DESCRIPTION
**Pull request by nicosvirjt**
## Description
Increased documentation for raise_error middleware. [Logger.md](https://github.com/lostisland/faraday/blob/main/docs/middleware/response/logger.md) already have this useful configuration line

## Todos
None

### CODE REVIEW

**Review by olleolleolle — APPROVED** [review]
❤️ This is lovely, thank you!

File: docs/middleware/response/raise_error.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ response returns with a 4xx or 5xx status code. All exceptions are initialized
 providing the response `status`, `headers`, and `body`.
 
 ```ruby
+conn = Faraday.new(url: 'http://sushi.com') do |faraday|
```
Comment (olleolleolle) [inline docs/middleware/response/raise_error.md:17]: We have moved to httpbingo URLs in most everywhere, can we change to that, here, too?

Comment (olleolleolle) [reply docs/middleware/response/raise_error.md:17]: https://github.com/lostisland/faraday/pull/1384 example here

Comment (nicosvirjt) [reply docs/middleware/response/raise_error.md]: Yea sure ! :)

**Review by olleolleolle — APPROVED** [review]
Thanks, merging!

**Pull request merged**